.zenodo.json

```
@@ -0,0 +1,162 @@
+{
+    "creators": [
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Vay, Jean-Luc"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Almgren, Ann"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Amorim, L\u00edgia Diana"
+      },
+      {
+        "affiliation": "Laboratoire d'Optique Appliqu\u00e9e",
+        "name": "Andriyash, Igor"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Belkin, Daniel"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Bizzozero, David"
```

@dbizzozero what is your ORCiD? :)
```suggestion
        "name": "Bizzozero, David",
       "orcid": "..."
```

---

ping @dbizzozero what is your ORCiD? :)

---

has no ORCiD